ipa-core/benches/oneshot/ipa.rs

```diff
@@ -155,6 +164,7 @@ async fn run(args: Args) -> Result<(), Error> {
     tracing::trace!("Preparation complete in {:?}", _prep_time.elapsed());
 
     let _protocol_time = Instant::now();
+    println!("about to run test_oprf_ipa");
```

Comment:
either a tracing::info, or lets remove this line as it seems to be for debug purposes only